ql/src/java/org/apache/hadoop/hive/ql/parse/TransformSpec.java

```diff
@@ -66,11 +66,63 @@ public void setTransformParam(Optional<Integer> transformParam) {
     this.transformParam = transformParam;
   }
 
+  public String transformTypeString() {
+    if (transformType == null) {
+      return null;
+    }
+    if (transformParam.isPresent()) {
+      return transformType.name() + "[" + transformParam.get() + "]";
```

Comment:
should it be "("?

Reply:
it won't match the value returned by Iceberg, there it is [

https://github.com/apache/iceberg/blob/4958663b4c863a44e4895ecf965fabf541083b39/api/src/main/java/org/apache/iceberg/transforms/Truncate.java#L160-L161